spec/spec_helper.rb

```diff
@@ -56,6 +56,17 @@ def with_random_region
 
 ActiveRecord::Migration.include(ActiveRecord::IdRegions::Migration)
 
+def suppress_migration_messages
+  save, ActiveRecord::Migration.verbose = ActiveRecord::Migration.verbose, false
+  yield
+ensure
+  ActiveRecord::Migration.verbose = save
+end
+
+def migration_versions
+  [5.2, 5.1, 5.0, 4.2].select { |v| ActiveRecord::VERSION::STRING >= v.to_s }
```

Comment:
This just allows us to run this test suite with a 5.0 or 5.1... where 5.2 version migration would not yet exist.